go/vt/vtgate/planbuilder/jointree.go

```diff
@@ -171,6 +253,46 @@ func (p parenTables) tableID() semantics.TableSet {
 	return res
 }
 
+func visitTables2(r relation, f func(tbl relation) (bool, error)) error {
```

Comment:
I this some secondary visitTables that must be done after calling `visitTables` at least once?

Reply:
oops. forgot to clean this up